frontend/apps/reader/modules/readerview.lua

```diff
@@ -792,7 +792,7 @@ function ReaderView:onReadSettings(config)
     self.highlight.saved = config:readSetting("highlight", {})
     -- Highlight formats in crengine and mupdf are incompatible.
     -- Backup highlights when the document is opened with incompatible engine.
-    local _page, page_highlights = next(self.highlight.saved)
+    local _, page_highlights = next(self.highlight.saved)
```

Comment:
This is going from readable to more cryptic :).
My suggestion `some_page_num, some_page_highlights` was indicating we are looking at "any" page for its first highlight.
If you don't like it, you can add a comment `-- Look at any first highlight among self.highlight.saved`
You can also instead of `_` just add `-- luacheck: no unused` at the end of the line (we have quite a few, mostly just for readability).